src/WebAPI/ClientApp/src/lang/de_DE.json

```diff
@@ -0,0 +1,416 @@
+					}
+				},
+				"date-and-time": {
+					"short-date-format": {
+						"label": "Datum kurzformat",
+						"title": "",
+						"text": ""
+					},
+					"long-date-format": {
+						"label": "Dazum langformat",
+						"title": "",
+						"text": ""
+					},
+					"time-zone": {
+						"label": "Zeitzone",
+						"title": "",
+						"text": ""
+					},
+					"time-format": {
+						"label": "Zeit Format",
+						"title": "",
+						"text": ""
+					},
+					"show-relative-dates": {
+						"label": "Zeige Relative Daten",
+						"title": "",
+						"text": ""
+					}
+				},
+				"confirmation-settings": {
+					"download-movie-confirmation": {
+						"label": "Frage nach Bestätigung für Filme Download",
+						"title": "Frage nach Bestätigung für Filme Download",
+						"text": ""
+					},
+					"download-tvshow-confirmation": {
+						"label": "Frage nach Bestätigung für Serien Download",
+						"title": "Frage nach Bestätigung für Serien Download",
+						"text": ""
+					},
+					"download-season-confirmation": {
+						"label": "Frage nach Bestätigung für Staffel Download",
+						"title": "Frage nach Bestätigung für Staffel Download",
+						"text": ""
+					},
+					"download-episode-confirmation": {
+						"label": "Frage nach Bestätigung für Folgen Download",
+						"title": "Frage nach Bestätigung für Folgen Download",
+						"text": ""
+					}
+				}
+			},
+			"advanced": {
+				"download-manager-section": {
+					"download-segments": {
+						"label": "Download Teile",
+						"title": "Download Teile",
+						"text": "PlexRipper Multithread-Downloads und kann Medien, die heruntergeladen werden sollen, in individuellen teilen herunterladen und wieder zusammen führen. Das führt in der regel zu einer höheren Downloadgesschwindigkeit aber benötigt auch etwas mehr Zeit um die Teile wieder zusammenzusetzen.\n\nDas setzen auf den Wert 1 deaktiviert die Funktion und lädt die Datei am Stück herunter. Die Anzahl der Segmente hat keinen Einfluss auf die Dauer des zusammen setzens."
+					}
+				},
+				"setup-section": {
+					"go-to-setup-page": {
+						"label": "gehe zur Setup-Seite",
+						"title": "gehe zur Setup-Seite",
+						"text": "Navigiert zur Setup-Seite."
+					}
+				},
+				"reset-db": {
+					"label": "Datenbank zurück setzen",
+					"title": "Datenbank zurück setzen",
+					"text": "Setzt die Datenbank zurück und erstellt eine Sicherung der alten Datenbank."
+				}
+			}
+		}
+	},
+	"confirmation": {
+		"skip-setup": {
+			"title": "Sind Sie sicher?",
+			"text": "Das überspringt das Setup und Sie müssen PlexRipper manuell einrichten."
+		},
+		"delete-account": {
+			"title": "Sind Sie sicher?",
+			"text": "This will delete this Plex account from Plex Ripper! \n\n This will also delete all inaccessible Plex shared servers, Plex libraries and all media that is contained within. Any active downloads that are started or queued with this account will be canceled and deleted. Completed downloads will remain untouched.",
```

Comment:
Dadurch wird der Plex-Account auf dem Plex Ripper gelöscht!\n\nEbenso werden alle nicht erreichbaren geteilten Plex-Server, Plex-Mediatheken und alle darin enthaltenen Medien gelöscht. Aktive Downloads, die mit diesem Account gestartet oder in die Warteschlange eingereiht wurden, werden abgebrochen und gelöscht. Abgeschlossene Downloads hingegen werden nicht angefasst.